webapp/.eslintrc.json

```diff
@@ -333,12 +333,8 @@
     ],
     "no-self-compare": 2,
     "no-sequences": 2,
-    "no-shadow": [
-      2,
-      {
-        "hoist": "functions"
-      }
-    ],
+    "no-shadow": "off",
+    "@typescript-eslint/no-shadow": ["error"],
```

Comment:
was this intentional?

Reply:
Yes, it's [known bug](https://github.com/typescript-eslint/tslint-to-eslint-config/issues/856) with linting typescript enums. Ideally we should update since it's been fixed but I am inclined to defer that to another time.